src/Drawer/Drawer.js

```diff
@@ -28,42 +29,42 @@ export const styleSheet = createStyleSheet('Drawer', (theme) => {
   };
 });
 
+type Props = {
+  /**
+   * The contents of the `Drawer`
+   */
+  children?: Object,
+  /**
+   * The CSS class name of the root element.
+   */
+  className?: string,
+  /**
+   * If set to true, the drawer will dock itself
+   * and will no longer slide in with an overlay
+   */
+  docked?: boolean,
+  open: boolean,
+  /**
+   * The CSS class name of the paper element.
+   */
+  paperClassName?: string,
+  zDepth: number,
```

Comment:
Yeah but what I mean is since we are using `react-docgen` to build the documentation, I wonder if without the `?`, it will display the prop as a required prop in docs which wouldn't be exactly accurate. A prop that has a default value to me should be displayed as optional.

Do you know what react-docgen's behavior is with this type of typing?